lib/Cake/Test/Case/View/Helper/HtmlHelperTest.php

```diff
@@ -553,6 +553,31 @@ public function testImageTagWithTheme() {
 		}
 	}
 
+	/**
+	 * testBase64ImageTag method
+	 *
+	 * @return void
+	 */
+	public function testBase64ImageTag()
+	{
+		$this->Html->request->webroot = '';
+		
+		$result = $this->Html->image('cake.icon.png', ['base64' => true]);
+		$this->assertTags($result, array(
+			'img' => array(
+				'src' => 'preg:/data:image\/png;base64,(?:[A-Za-z0-9+\/]{4})*(?:[A-Za-z0-9+\/]{2}==|[A-Za-z0-9+\/]{3}=|[A-Za-z0-9+\/]{4})/',
+				'alt' => ''
+			)));
+
+		$result = $this->Html->image('/img/cake.icon.png', ['base64' => true]);
+		$this->assertTags($result, array(
+			'img' => array(
+				'src' => 'preg:/data:image\/png;base64,(?:[A-Za-z0-9+\/]{4})*(?:[A-Za-z0-9+\/]{2}==|[A-Za-z0-9+\/]{3}=|[A-Za-z0-9+\/]{4})/',
+				'alt' => ''
+			)));
+
+	}
```

Comment:
Ok, but in the HTML Helper there's only one exception thrown, and that's if it can't load the config for the tags definition. No one will expect the `link()` function to throw, and the rule of least disruption is for it to just echo out an empty image.

That said, I'm happy to make it throw if we feel that's a better option. Throw by default might be a better choice in 3.x?